docs/roadmap.md

```diff
@@ -19,12 +19,33 @@ The following roadmap gives an overview of what we are currently working on and
 
 *Status*: tech spec out
 
-#### Integrate with data quality system
+#### Integrate with Data Quality system
 *What*: Integrate with different data quality systems to provide quality score.
 
 *Status*: planning
 
 ## Mid Term - Our Next steps
+#### Improve search ranking
+*What*: Update search ranking to be informed by "badges" that may exist on data sets e.g. deprecated, etc.
+
+*Status*: planning
+
+#### Notifications when a table evolves
```

Comment:
we do have a notification system so far with jira / email. Could you share more on the difference?

Reply:
This is around getting notifications in Amundsen itself. I remember Ryan (Lyft's intern) worked on this half.

User story here is that when a new column gets added in a bookmarked table, or if there's a delay in the table, the consumer would get notification.